package.json

```diff
@@ -48,48 +48,50 @@
     "generate-docs": "node_modules/.bin/jsdoc -c jsdoc.json"
   },
   "dependencies": {
-    "@mojaloop/central-services-auth": "0.2.0",
-    "@mojaloop/central-services-database": "3.1.0-snapshot",
-    "@mojaloop/central-services-error-handling": "3.7.0-snapshot",
-    "@mojaloop/central-services-metrics": "5.1.0",
-    "@mojaloop/central-services-shared": "4.1.0-snapshot",
-    "@mojaloop/central-services-stream": "4.1.0-snapshot",
-    "@mojaloop/forensic-logging-client": "0.5.1",
-    "@now-ims/hapi-now-auth": "1.3.0",
+    "@mojaloop/central-services-auth": "5.2.1",
+    "@mojaloop/central-services-database": "5.2.1",
+    "@mojaloop/central-services-error-handling": "5.2.0",
+    "@mojaloop/central-services-metrics": "5.2.0",
+    "@mojaloop/central-services-shared": "5.2.0",
+    "@mojaloop/central-services-stream": "5.2.0",
+    "@mojaloop/forensic-logging-client": "5.2.0",
+    "@now-ims/hapi-now-auth": "1.3.1",
     "blipp": "3.1.2",
     "bluebird": "3.5.3",
     "boom": "7.3.0",
+    "catbox": "10.0.6",
+    "catbox-memory": "4.0.1",
     "commander": "2.19.0",
-    "docdash": "1.0.2",
+    "docdash": "1.0.3",
     "flat": "4.1.0",
     "glob": "7.1.3",
     "good": "8.1.2",
-    "hapi": "17.6.0",
+    "hapi": "18.1.0",
     "hapi-auth-basic": "5.0.0",
-    "hapi-auth-bearer-token": "6.1.0",
-    "hapi-swagger": "9.0.2",
-    "immutable": "3.8.2",
-    "inert": "5.1.0",
+    "hapi-auth-bearer-token": "6.1.1",
+    "hapi-swagger": "9.3.1",
+    "immutable": "4.0.0-rc.12",
```

Comment:
I'm a little concerned about using a release candidate version. is there a close latest version?

Reply:
![image](https://user-images.githubusercontent.com/20338055/53424485-f8749b00-39eb-11e9-932d-41e36cfd5a9d.png)
Agreed - reverted it back to 3.8.2